app/assets/javascripts/backbone/views/user_list_view.js.coffee

```diff
@@ -0,0 +1,21 @@
+class ELMO.Views.UserListView extends Backbone.View
+
+  el: '.link_set'
+
+  events:
+    'click .list-groups': 'show_groups_modal'
+
+  initialize: (params) ->
+    this.params = params
+
+  show_groups_modal: (event) ->
+    event.preventDefault()
+    ELMO.app.loading(true)
+    $.ajax
+      url: ELMO.app.url_builder.build('user_groups'),
+      method: "get",
+      data: {},
```

Comment:
I think you can leave this out. You also can remove the trailing commas. Coffee adds them based on indent.